packages/@react-stately/data/docs/useAsyncList.mdx

```diff
@@ -174,6 +174,54 @@ let list = useAsyncList({
 });
 ```
 
+## Filtering
+
+There are many instances where your list of data may need to be filtered, such as during user lookup or query searches.
+For server side filtering, this can be implemented by leveraging the `filterText` passed to the `load` as an API parameter.
+`useAsyncList` returns the state variable `filterText` and its setter `setFilterText`, allowing you to update the
+`filterText` used within `load` in response to any updates within your component.
+
+### Server side filtering
+
+The example below shows how server side filtering could be implemented by using `filterText` in the `load` function and passing a parameter to the API.
+The input text of the ComboBox is controlled by providing `list.filterText` as the ComboBox's `inputValue` prop. By passing `list.setFilterText` to
+the ComboBox via the `onInputChange` prop, the `filterText` within `load` is updated whenever the end user modifies the ComboBox's
```

Comment:
I don't think we need to describe `setFilterText` again since we already explained it above.